protocol/implementations/js/src/builders.ts

```diff
@@ -0,0 +1,36 @@
+import { MessageTypes, TbDEXMessage } from './types.js'
+import {ulid} from 'ulidx'
+
+// maybe extract this somewhere?
+const ID_NAMESPACE = 'tbdex'
+
+function createId(messageType: string): string {
+  return `${ID_NAMESPACE}:${messageType}:${ulid()}`
```

Comment:
temp implementation while the cbor/id requirements conversation continues: https://sq-tbd.slack.com/archives/C05B492CW1H/p1687222735830969